frontend/src/react/OpBlockNoteContainer.tsx

```diff
@@ -47,8 +47,16 @@ const schema = BlockNoteSchema.create({
   },
 });
 
+const detectTheme = (): "light" | "dark" => {
+  if (document.body.getAttribute('data-color-mode') === 'dark') {
+    return 'dark';
+  }
+  return 'light';
+};
+
 export default function OpBlockNoteContainer({ inputField, inputText }: OpBlockNoteContainerProps) {
   const [isLoading, setIsLoading] = useState(true);
+  const [theme, setTheme] = useState<"light" | "dark">(detectTheme);
```

Comment:
Would be nice to define a type here for the light/dark

```typescript
type OpTheme = "light"|"dark"
...
const [theme, setTheme] = useState<OpTheme>(detectTheme);
```

bear with me that I did not check this on an editor, so syntax might be wrong